step-ktlint-check-diff.yml

```diff
@@ -0,0 +1,25 @@
+# Downloads and runs ktlint on changed files.
+
+parameters:
+- name: version
+  type: string
+  default: "0.37.2"
```

Comment:
Thinking out loud: What if we removed the default value ? This would put more "responsibility" on the user of this template but would make it less volatile and easier to maintain? What's your opinion?

Reply:
I also thought about it, but the script would work without it as well (because Pipelines would use an empty string as the default). So it would only fail once it's running (because the url won't exist), which seems like a bad out-of-the-box experience.

We're using defaults in other templates as well, I think for a similar reason. Having the default also works nicely as an example (i.e. what format to use). But I also agree with your points! :)